openedx/features/course_experience/utils.py

```diff
@@ -127,7 +127,6 @@ def mark_last_accessed(user, course_key, block):
         'html',
         'problem',
         'video',
-        'discussion',
```

Comment:
I think this could present another issue when a discussion block is the last block that was marked complete for a user in a course, due to lines 81-82 above.  I would leave this line in and change line 94 above to something like:
```python
completable_blocks = [child for child in block['children'] if child['type'] != 'discussion']
if len([child['complete'] for child in block['children'] if child['complete']]) == len(completable_blocks):
    block['complete'] = True
```